packages/remix-dev/__tests__/create-test.ts

```diff
@@ -275,16 +275,10 @@ describe("the create command", () => {
         "\n\n" +
         getSuccessMessage(path.join("<TEMP_DIR>", "template-to-js"))
     );
-    expect(fse.existsSync(path.join(projectDir, "package.json"))).toBeTruthy();
-    expect(fse.existsSync(path.join(projectDir, "app/root.jsx"))).toBeTruthy();
     expect(fse.existsSync(path.join(projectDir, "app/root.tsx"))).toBeFalsy();
+    expect(fse.existsSync(path.join(projectDir, "app/root.jsx"))).toBeTruthy();
     expect(fse.existsSync(path.join(projectDir, "tsconfig.json"))).toBeFalsy();
     expect(fse.existsSync(path.join(projectDir, "jsconfig.json"))).toBeTruthy();
-    expect(fse.existsSync(path.join(projectDir, "app/utils.js"))).toBeTruthy();
-    let pkgJSON = JSON.parse(
-      fse.readFileSync(path.join(projectDir, "package.json"), "utf-8")
-    );
-    expect(Object.keys(pkgJSON.devDependencies)).not.toContain("typescript");
```

Comment:
Made this test smaller, as we're already testing this in the migration test itself.